yarn.lock

```diff
@@ -89,10 +89,17 @@
     "@babel/helper-annotate-as-pure" "^7.12.13"
     regexpu-core "^4.7.1"
 
+<<<<<<< HEAD
```

Comment:
Looks like the rebase of yarn.lock did not work well. You can probably take out the yarn.lock update, as I update it already (see dedicated commit).

Reply:
I did not save the file :( its all fine now

Reply:
Well, it still contains the broken merges in the early commits, but bi-secting is not really a requirement for this repo and we can also squash everything once the PR is finished.